src/main/typescript/wcardinal/ui/d-view.ts

```diff
@@ -185,7 +185,7 @@ export interface DView {
 	zoomOut(duration?: number, stop?: boolean): void;
 
 	/**
-	 * Zooms into the spcecified position.
+	 * Zooms into the specific position.
```

Comment:
Thank you for the PR.
Regarding lines 188 and 236, I think they should be something like
```suggestion
	 * Zooms into the specified position.
```
or 
```suggestion
	 * Zooms into the given position.
```

Any thoughts?

Reply:
I prefer "given" in this case.
For me, specified hints there is a process to figure out or choose something, which actually does not exist here.
Anyway, not a native speaker. Just my opinion.

Reply:
> Anyway, not a native speaker. Just my opinion.

Me neither :)

For example, `Map#containsKey`'s JavaDoc uses the word `specified`. So both `specified` and `given` are equally appropriate I guess.

https://docs.oracle.com/javase/7/docs/api/java/util/AbstractMap.html#containsKey(java.lang.Object)

So please change to `given position`. Then I will merge your PR.

Reply:
OK. Changed to "given position".

> For example, Map#containsKey's JavaDoc uses the word specified. So both specified and given are equally appropriate I guess.
My personal guess is that, for key or enum, specified is more appropriate because we are choosing something among several options.
I don't know it is correct or not.

Reply:
> > For example, Map#containsKey's JavaDoc uses the word specified. So both specified and given are equally appropriate I guess.
> 
> My personal guess is that, for key or enum, specified is more appropriate because we are choosing something among several options. I don't know it is correct or not.

Yea, might be that is the reason why `specified` is used there.

> OK. Changed to "given position".

Thank you for the changes.